saleor/graphql/order/types.py

```diff
@@ -155,33 +221,42 @@ def resolve_invoice_number(root: models.OrderEvent, _info):
         return root.parameters.get("invoice_number")
 
     @staticmethod
-    def resolve_lines(root: models.OrderEvent, _info):
+    def resolve_lines(root: models.OrderEvent, info):
         raw_lines = root.parameters.get("lines", None)
 
         if not raw_lines:
             return None
 
         line_pks = []
         for entry in raw_lines:
-            line_pks.append(entry.get("line_pk", None))
-
-        lines = models.OrderLine.objects.filter(pk__in=line_pks).all()
-        results = []
-        for raw_line, line_pk in zip(raw_lines, line_pks):
-            line_object = None
-            for line in lines:
-                if line.pk == line_pk:
-                    line_object = line
-                    break
-            results.append(
-                OrderEventOrderLineObject(
-                    quantity=raw_line["quantity"],
-                    order_line=line_object,
-                    item_name=raw_line["item"],
+            line_pk = entry.get("line_pk", None)
+            if not line_pk:
+                continue
+            line_pks.append(line_pk)
```

Comment:
Minor stylistic change but I think this is more intuitive:
```suggestion
            if line_pk:
                line_pks.append(line_pk)
```

Reply:
Good catch. I was suggesting with the old code too much! thanks